sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyDetectionConfigurationLiveTests.cs

```diff
@@ -24,9 +28,11 @@ public AnomalyDetectionConfigurationLiveTests(bool isAsync) : base(isAsync)
         public async Task CreateAndGetDetectionConfigurationWithHardCondition(bool useTokenCredential)
         {
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient(useTokenCredential);
+            await using DisposableDataFeed disposableDataFeed = await CreateDisposableDataFeedAsync(adminClient);
 
             string configName = Recording.GenerateAlphaNumericId("config");
-            var description = "This hook was created to test the .NET client.";
+            string metricId = disposableDataFeed.DataFeed.MetricIds[MetricName];
```

Comment:
We were using a hardcoded Metric ID before (obtained from the data feed we were using from our static resource). Now we're getting the ID from the newly created mocked data feed.